metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -57,19 +56,11 @@ import org.eclipse.{lsp4j => l}
  *   // ...
  * }}}
  *
- * Includes the following converters from the standard library: {{{
- *  import scala.compat.java8.FutureConverters._
- *  import scala.meta.internal.jdk.CollectionConverters._
- * }}}
- *
  * If this doesn't scale because we have too many unrelated extension methods
  * then we can split this up, but for now it's really convenient to have to
  * remember only one import.
  */
-object MetalsEnrichments
-    extends DecorateAsJava
```

Comment:
> This was usually added via MetalsEnrichments and the DecorateAsScala/DecorateAsJava traits, I think we can just switch to AsJavaExtensions and AsScalaExtensions instead.

Actually here, we could just replace those I think.

https://www.scala-lang.org/api/current/scala/collection/convert/AsJavaExtensions.html